### ISSUE TITLE
App Crashes when using onSwipe viewModifier using iphone8/8plus & iphone11 on iOS 13.0-13.1.3

### ISSUE THREAD

**Issue by MichaelNeas** [EnesKaraosman/SwipeCell#2]
Hey! 

Love the package, talked to you briefly on slack a while back.

Unfortunately I had a couple users hard crash my application on their iphone 11 with iOS 13.0-13.1.3

I don't even think this is a problem with your package but **all** custom view modifiers on that device with that os version. Here's an image of a super basic use case in a fresh application to prove.

```
import SwiftUI
import SwipeCell

struct ContentView: View {
    @State var test = [1,2,3,4,5]
    var body: some View {
        List(test, id: \.self) { num in
            Text(num.description)
                .onSwipe()
        }
    }
}
```
I've tried a bunch of variations of the above code and it always crashes 😭 

![Screen Shot 2020-11-11 at 7 04 55 PM](https://user-images.githubusercontent.com/10774848/98878439-003ed200-2451-11eb-8bdd-bc9dc27aed47.png)

**Comment by EnesKaraosman** [comment]
Let me download iOS13.1 simulator first :)

**Comment by EnesKaraosman** [comment]
wow it was because of `AnimatableModifier`, when I switch back to `ViewModifier` it worked.

**Comment by EnesKaraosman** [comment]
So @MichaelNeas  thanks for letting me know, I fixed it, please use version `0.1.2` .
Bug was nonsense because if you investigate core interface, you'll se that `AnimatableModifier` is actually <br> `public protocol AnimatableModifier : Animatable, ViewModifier { }` <br>
`public struct SlidableModifier: ViewModifier, Animatable { }` works but below not works :D <br>
`public struct SlidableModifier: AnimatableModifier { }`

**Comment by MichaelNeas** [comment]
@EnesKaraosman wowww that's interesting and bizarre! Thank you for checking this out 😄 I'll update today!
